src/armor_layers.cpp

```diff
@@ -175,12 +175,15 @@ void player::sort_armor()
             } else {
                 mvwprintz(w_sort_middle, cont_h - 12 + i, 2, c_ltgray, "%s:", armor_cat[i].c_str());
             }
-            mvwprintz(w_sort_middle, cont_h - 12 + i, middle_w - 16, c_ltgray, "%d+%d = ", armorenc,
-                      enc - armorenc);
+            true_enc = enc - armorenc;
+            char my_spaces[]    = "   ";    // lol
+            char *spaces        = (char*)&my_spaces;
```

Comment:
C++ is weird to me coming from ASM & C, lol. Also, that was an overlook on my part, as far as char** is concerned, I just simply wasn't getting the correct pointer's pointer. Still get fumbled sometimes on pointer pointers, as far as when to grab them.